packages/react-datetime/src/components/DatePicker/DatePicker.tsx

```diff
@@ -3,165 +3,108 @@ import { css } from '@patternfly/react-styles';
+  dateParse?: (value: string) => Date;
   /** Flag indicating the date picker is disabled*/
   isDisabled?: boolean;
-  /** Specify the locale of the date. */
-  locale?: Locale;
   /** String to display in the empty date picker field as a hint for the expected date format */
   placeholder?: string;
-  /** A date string. Defaults to the current date. */
+  /** Value of TextInput */
   value?: string;
-  /** Error message to display when the date is provided in an invalid format. */
-  invalidFormatErrorMessage?: string;
-  /** Error message to display when the date is provided in outside the valid date range. */
-  dateOutOfRangeErrorMessage?: string;
-  /** Error message to display when the date is provided before the provided minDate. */
-  beforeMinDateErrorMessage?: string;
-  /** Error message to display when the date is provided after the provided maxDate. */
-  afterEndDateErrorMessage?: string;
+  /** Error message to display when the TextInput cannot be parsed. */
+  invalidFormatText?: string;
   /** Callback called every time the input value changes */
   onChange?: (value: string, date?: Date) => void;
   /** Text for label */
   helperText?: React.ReactNode;
-  /** Aria label for the button */
+  /** Aria label for the button to open the date picker */
   buttonAriaLabel?: string;
   /** The element to append the popover to */
   appendTo?: HTMLElement | ((ref?: HTMLElement) => HTMLElement);
   /** Props to pass to the Popover */
   popoverProps?: Omit<PopoverProps, 'appendTo'>;
+  /** How to format months in Calendar */
+  monthFormat?: (date: Date) => React.ReactNode;
+  /** How to format week days in Calendar */
+  weekdayFormat?: (date: Date) => React.ReactNode;
+  /** How to format days in Calendar */
+  longWeekdayFormat?: (date: Date) => React.ReactNode;
+  /** How to format days in Calendar */
+  buttonFormat?: (date: Date) => React.ReactNode;
+  /** If using the default formatters which locale to use. Undefined defaults to current locale. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation */
+  locale?: string;
+  /** Day of week that starts the week in Calendar. 0 is Sunday, 6 is Saturday. */
+  weekStart?: 0 | 1 | 2 | 3 | 4 | 5 | 6;
+  /** Functions that returns an error message if a date is invalid */
+  validators?: ((date: Date) => string)[];
 }
 
+const isValidDate = (date: Date) => !isNaN(date.getTime());
+
 export const DatePicker: React.FunctionComponent<DatePickerProps> = ({
   className,
-  isInvalid = false,
-  minDate: minDateProp = '',
-  maxDate: maxDateProp = '',
-  dateFormat = 'MM/dd/yyyy',
+  locale = undefined,
+  dateFormat = (date: Date) => date.toISOString().substring(0, 10),
+  dateParse = (val: string) => new Date(`${val}T00:00:00`),
   isDisabled = false,
-  locale = Locales.enUS,
-  placeholder = 'MM/dd/yyyy',
-  value: valueProp = '',
+  placeholder = 'yyyy-MM-dd',
+  value: valueProp = dateFormat(new Date()),
```

Comment:
I think it should be empty rather than have it default to the current date if no default date is provided.

Reply:
Done.